simpleflow/swf/executor.py

```diff
@@ -18,10 +18,15 @@
     task,
 )
 from simpleflow.activity import Activity
+from simpleflow.base import Submittable
 from simpleflow.history import History
 from simpleflow.swf import constants
 from simpleflow.swf.helpers import swf_identity
 from simpleflow.swf.task import ActivityTask, WorkflowTask
+from simpleflow.task import (
```

Comment:
(Replying to myself: not so clean, as we've got `base` already; perhaps one day cleanup the structure while keeping the historical namespaces)